src/intl/en/page-learn.json

```
@@ -50,7 +50,7 @@
   "defi-card-description": "Explore an alternative financial system that is built without banks and is open to anyone.",
   "defi-card-button": "What is DeFi?",
   "stablecoins-card-title": "Stablecoins",
-  "stablecoins-card-description": "Stablecoins are cryptocurrencies designed to stay at a fixed value of a dollar.",
+  "stablecoins-card-description": "Stablecoins are cryptocurrencies that attempt to peg their value to a currency, commodity, or some other financial instrument.",
```

```suggestion
  "stablecoins-card-description": "Stablecoins are cryptocurrencies designed to peg their value to a currency, commodity, or some other financial instrument.",
```

Upon reading this again, I think this is a bit better. Otherwise, good to go :)